.github/workflows/ci.yml

```
@@ -11,7 +11,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macOS-latest]
-        node-version: [14.x, 16.x]
+        node-version: [16.x]
```

Might as well add Node 18 now too.

```suggestion
        node-version: [16.x, 18.x]
```